docs/configure-rest.md

```diff
@@ -310,8 +311,8 @@ of these parameters can be defined in the Harvest or object configuration files
-| - `counter`        | duration (Go-syntax)           | poll frequency of updating the counter metadata cache                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | 20 minutes |
-| - `instance`       | duration (Go-syntax)           | poll frequency of updating the instance cache                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | 10 minutes |
+| - `counter`        | duration (Go-syntax)           | poll frequency of updating the counter metadata cache                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |   24 hours |
+| - `instance`       | duration (Go-syntax)           | poll frequency of updating the instance cache.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |  720 hours |
```

Comment:
Since the instance schedule is only used by workload and workload_volume we should mention it here or remove it altogether since it is highlighted in `conf/restperf/default.yaml`